src/xcode/ENA/ENA/Source/View Helpers/AppStrings.swift

```diff
@@ -2123,6 +2123,17 @@ enum AppStrings {
 				static let defaultAlertRetryButton = NSLocalizedString("HealthCertifiedPerson_DefaultAlert_RetryButton", comment: "")
 
 				static let title = NSLocalizedString("HealthCertifiedPerson_UpdateConsent_Title", comment: "")
+
+				static let deleteNotice = NSLocalizedString("HealthCertifiedPerson_UpdateConsent_Delete_Notice", comment: "")
+				static let cancelNotice = NSLocalizedString("HealthCertifiedPerson_UpdateConsent_Cancel_Notice", comment: "")
+
+				static let legalTitle = NSLocalizedString("HealthCertifiedPerson_UpdateConsent_Legal_Title", comment: "")
+				static let legalSubtitle = NSLocalizedString("HealthCertifiedPerson_UpdateConsent_Legal_Subtitle", comment: "")
+				static let legalBullet1 = NSLocalizedString("HealthCertifiedPerson_UpdateConsent_Legal_Bullet_1", comment: "")
+				static let legalBullet2 = NSLocalizedString("HealthCertifiedPerson_UpdateConsent_Legal_Bullet_2", comment: "")
+
+				static let bulletPoint_1 = NSLocalizedString("HealthCertifiedPerson_UpdateConsent_Bullet_Point_1", comment: "")
+				static let bulletPoint_2 = NSLocalizedString("HealthCertifiedPerson_UpdateConsent_Bullet_Point_2", comment: "")
```

Comment:
Here too, have a look into #4265, perhaps merge the branch into yours to get all refs.

Reply:
Solved